### PR TITLE
Add Novu

### PR DESCRIPTION
**Pull request by fmerian**
https://novu.co/

## checklist

Please make sure you've complied with the [contribution guidelines](https://github.com/agamm/awesome-developer-first/blob/main/CONTRIBUTING.md):
- [x] The homepage clearly states that the product is for developers.
  > "Headless", "API-First", "SaaS" are frequently used keywords.
  > Usually this means that the front page has some code examples :)
- [x] The product is available now and requires payment.
- [x] The PR has a descriptive title -- e.g., `Add {Product}`, not `Update README.md`.
- [x] The new submission isn't a duplicate.
- [x] The entry has a description, is sorted alphabetically, and ends with a full stop.

Thanks!

### CODE REVIEW

**Review by agamm — COMMENTED** [review]
What does "fully functional" mean in this context?

**Comment by fmerian** [comment]
> What does "fully functional" mean in this context?

probably unnecessary

**Comment by agamm** [comment]
"embedded" ? (sorry for being picky)

**Comment by fmerian** [comment]
simplified description to:
```
Open-source notification infrastructure.
```
thoughts @agamm?

File: README.md [hunk 1]
```diff
@@ -234,6 +234,7 @@ For the latest additions [click here](https://github.com/agamm/awesome-developer
 * [Applozic](https://www.applozic.com/) - Chat SDKs, real-time messaging.
 * [Knock](https://knock.app) - Notifications as a service.
 * [MagicBell](https://www.magicbell.com/) - Real-time notification system with API & UI components.
+* [Novu](https://novu.co/) - Open-source notification infrastructure.
```
Comment (agamm) [inline README.md:237]:
```suggestion
* [Novu](https://novu.co/) - Open-source notification (Email, SMS, Direct, and Push) management.
```

**Review by agamm — CHANGES_REQUESTED** [review]
WDYT?

**Review by agamm — APPROVED** [review]
:+1:

**Pull request merged**